.babelrc

```diff
@@ -0,0 +1,16 @@
+{
+  "presets": [
+    ["@babel/preset-env", {
+      "targets": {
+        "browsers": [
+          "last 2 versions",
+          "IE 11"
+        ]
+      }
+    }]
+  ]
+//  "plugins": [
```

Comment:
I think it's better to remove this if you doesn't use it.

Reply:
Removed.